chirp/drivers/radtel_t18.py

```diff
@@ -376,7 +376,7 @@ def model_match(cls, data):
     """Match the opened/downloaded image to the correct version"""
 
     if len(data) == cls._memsize:
-        rid = data[0x03D0:0x03D8]
+        rid = data[0x03D0:0x03D8].decode('ascii', 'replace')
         return "P558" in rid
```

Comment:
I think we should not decode to ascii here and instead check for ```b"P558" in rid``` right? We've done that everywhere else, AFAIK, why be different here?